Sources/Defaults/Observation+Combine.swift

```diff
@@ -151,4 +151,24 @@ extension Defaults {
 		return combinedPublisher
 	}
 }
+
+@available(macOS 10.13, OSXApplicationExtension 10.13, watchOSApplicationExtension 4.0, *)
+extension Defaults.NSSecureCodingKeyChange: Equatable where Value: Equatable {
+	public static func == (lhs: Self, rhs: Self) -> Bool {
+		lhs.newValue == rhs.newValue
```

Comment:
While this works for your use-case, I don't think this is correct. It should be equatable on all the properties. You can just rely on compiler synthesis for that.

Reply:
@sindresorhus got it, fixed.